forge/ee/db/controllers/DeviceGroup.js

```diff
@@ -214,6 +243,58 @@ module.exports = {
         await app.db.models.Device.update({ DeviceGroupId: null }, { where: { id: deviceIds.removeList, DeviceGroupId: deviceGroup.id }, transaction })
     },
 
+    // updateSettings: async function (app, deviceGroup, settings, user) {
```

Comment:
Unused code? Something for the future?

Reply:
Ah, that slipped through. Originally, I wrote this as a separate function but the tests revealed I need to restructure.

Will remove.

Reply:
resolved in [Remove unused code](https://github.com/FlowFuse/flowfuse/pull/4659/commits/f2aad783993211f47a918dd82d14a3951d8f181d)